### ISSUE TITLE
Change own password

### ISSUE THREAD

**Issue by jaegeral** [google/timesketch#605]
Having multiple users on a timesketch system it would be handy to change the own password

**Comment by adamjnichols** [comment]
For now, most user admin tasks are accomplished via tsctl.  I don't think we plan to change this for the foreseeable future, but at some point we would like to have some more robust user profiles where this could be possible.

Thanks for the suggestion!

**Comment by kiddinn** [comment]
#159 does have the request to add more user profiles, there would the option to change one password lie.

**Comment by honorthecode** [comment]
Agreed, this would be extremely useful, especially if it could be done in the UI.

**Comment by kiddinn** [comment]
This will be added in the API and API Client here: #1700

Missing is the UI component.
